src/test/resources/application-test.yml

```diff
@@ -1,3 +1,35 @@
+spring:
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          public-key-location: classpath:local-public-key.pub
+
+  datasource:
+    url: 'jdbc:h2:mem:afer-db;MODE=PostgreSQL'
```

Comment:
we might want to run tests against a real postgres server eventually but this makes a sensible default to start with 👍